pkg/cmd/prune-backups/prune_test.go

```diff
@@ -54,15 +54,15 @@ func TestRetainByNumber(t *testing.T) {
 		time.Sleep(1 * time.Second)
 	}
 
-	require.NoError(t, retainByNumber(6))
```

Comment:
would you mind adding some tests that will test varying folders instead of hardcoding BasePath?

Reply:
what also works for me: we do not have to change BasePath anymore, we can simply inject the tmp folder into the arguments